lib/Components/Header/header.css

```diff
@@ -0,0 +1,82 @@
+/* HEADER STARTS */
+
+.sm-header{
+    grid-area: header;
+    padding: 0 1rem;
+    position: fixed;
+    z-index:2;
+    width: 100%;
+    top:0;
+    background: var(--COLOR-BG); 
+    margin: auto;
+    border-bottom: 2px solid var(--COLOR-HEADER-FONT-DARK);
+}
+
+.sm-nav-new{
+    padding:1rem;
+}
+
+#sc-login{
+    white-space: nowrap;
+}
+
+#sc-logo {
+    white-space: nowrap;
+    display: inline-block;
+}
+
+.btn-social{
+    outline: none;
+    border:none;
+    background: none;
```

Comment:
Suggestion: Use consistent use of selectors, if you are using everywhere classes, use classes.

Reply:
okay